Ubuntu1804-readme.md

```diff
@@ -1 +1,297 @@
+- Node.js 12.18.3
+- Python 2.7.17
+- Python3 3.6.9
+- Powershell 7.0.3
+- Ruby 2.5.1p57
+- Swift 5.2.5
+- Julia 1.5.0
+
+### Package Management
+- Homebrew 2.4.13
+- Gem 3.1.4
+- Miniconda 4.8.3
+- Helm 3.3.0
+- NPM 6.14.6
+- Yarn 1.22.4
+- pip 9.0.1 (python 2.7)
+- pip3 9.0.1 (python 3.6)
+- Vcpkg 2020.06.15
+
+### Project Management
+- Ant 1.10.5
+- Gradle 6.6.1
+- Maven 3.6.3
+- sbt 1.3.13
+
+### Tools
+- 7-Zip 16.02
+- Ansible 2.9.12
+- AzCopy7 (available by azcopy alias) 7.3.0
+- AzCopy10 (available by azcopy10 alias) 10.6.0
+- Bazel 3.4.1
+- Bazelisk 1.6.1
+- Buildah 1.15.1
+- CMake 3.17.0
+- Docker Compose 1.26.2
+- Docker-Moby 19.03.12
+- Docker-Buildx 0.4.2
+- Git 2.28.0
+- Git LFS 2.11.0
+- Git-ftp 1.3.1
+- Google Cloud SDK 307.0.0
+- Haveged 1.9.1
+- Heroku 7.42.10
+- HHVM (HipHop VM) 4.71.0
+- Podman 2.0.4
+- Skopeo 1.1.1
+- svn 1.9.7
+- Kustomize 3.8.1
+- Kind 0.8.1
+- Kubectl 1.18.8
+- Minikube 1.12.3
+- Mercurial 4.5.3
+- MongoDB 4.4.0
+- Leiningen 2.9.4
+- Newman (5.1.2)
+- nvm 0.35.3
+- Packer 1.6.1
+- PhantomJS 2.1.1
+- PostgreSQL 12.4
+- Swig 3.0.12
+- Terraform 0.13.0
+
+
+### CLI Tools
+- Azure CLI (azure-cli) 2.10.1
+- Azure CLI (azure-devops) 0.18.0
+- Alibaba Cloud CLI 3.0.56
+- AWS CLI 1.18.125
+- AWS CLI Session manager plugin 1.1.61.0
+- AWS SAM CLI 1.1.0
+- Hub CLI 2.14.2
+- GitHub CLI 0.11.1
+- Netlify CLI 2.59.1
+- oc CLI 4.5.0
+- ORAS CLI 0.8.1
+- Vercel CLI 20.0.0
+
+### Java
+| Version   | Vendor       | Environment Variable |
+| --------- | ------------ | -------------------- |
+| 1.7.0_272 | Zulu         | JAVA_HOME_7_X64      |
+| 1.8.0_265 | AdoptOpenJDK | JAVA_HOME_8_X64      |
+| 11.0.8    | AdoptOpenJDK | JAVA_HOME_11_X64     |
+| 12.0.2    | AdoptOpenJDK | JAVA_HOME_12_X64     |
+
+### PHP
+| Tool | Versions |
+|-|-|
+| PHP | 7.1.33<br>7.2.33<br>7.3.21<br>7.4.9 |
+| Composer | 1.10.10 |
+| PHPUnit | 7.5.20 |
+
+### Haskell
+- GHC 8.10.2
+- Cabal 3.2.0.0
+- Stack 2.3.3
+
+### Rust Tools
+- Rust 1.45.2
+#### Packages
+- rustup 1.22.1
+- rustdoc 1.45.2
+- bindgen 0.55.1
+- cbindgen 0.14.4
+- cargo (1.45.1)
+- cargo audit 0.12.0
+- clippy 0.0.212
+- cargo outdated 0.9.11
+- rustfmt 1.4.17
+
+### Browsers and webdrivers
+- Google Chrome 85.0.4183.83
+- ChromeDriver 85.0.4183.38
+- Mozilla Firefox 79.0
+- Geckodriver 0.27.0
+- Selenium server standalone (available via SELENIUM_JAR_PATH environment variable)
+
+### .NET Core SDK
+- 2.1.300 2.1.301 2.1.302 2.1.401 2.1.402 2.1.403 2.1.500 2.1.502 2.1.503 2.1.504 2.1.505 2.1.506 2.1.507 2.1.508 2.1.509 2.1.510 2.1.511 2.1.512 2.1.513 2.1.514 2.1.515 2.1.516 2.1.517 2.1.602 2.1.603 2.1.604 2.1.605 2.1.606 2.1.607 2.1.608 2.1.609 2.1.610 2.1.611 2.1.612 2.1.613 2.1.614 2.1.700 2.1.701 2.1.801 2.1.802 2.1.803 2.1.804 2.1.805 2.1.806 2.1.807 2.1.808 2.1.809 3.1.100 3.1.101 3.1.102 3.1.103 3.1.104 3.1.105 3.1.106 3.1.107 3.1.200 3.1.201 3.1.202 3.1.300 3.1.301 3.1.302 3.1.401
+
+### Az Module
+- 1.0.0 1.6.0 2.3.2 2.6.0 2.8.0 3.1.0 3.5.0 3.8.0 4.3.0 4.4.0
+
+### MySQL
```

Comment:
also can be added to the separate DBs section

Reply:
Done